socorro/external/postgresql/products.py

```diff
@@ -120,24 +120,3 @@ def get(self, **kwargs):
             'hits': results,
             'total': len(results),
         }
-
-    def post(self, **kwargs):
-        """adding a new product"""
-        filters = [
-            ("product", None, "str"),
-            ("version", None, "str"),
-        ]
-        params = external_common.parse_arguments(filters, kwargs)
-        with self.get_connection() as connection:
-            try:
-                result, = single_row_sql(
-                    connection,
-                    "SELECT add_new_product(%s, %s)",
-                    (params['product'], params['version']),
-                )
-            except psycopg2.Error:
-                connection.rollback()
-                return False
-            else:
-                connection.commit()
-            return result
```

Comment:
Let's keep this for now. It'll make it easier for me to move the `product` table over in the future.